frontend/components/StatusIndicator/_styles.scss

```diff
@@ -2,10 +2,7 @@
   display: flex;
   align-items: center;
   color: $core-fleet-blue;
-
-  ::first-letter {
-    text-transform: capitalize;
-  }
+  text-transform: capitalize;
```

Comment:
Does this conflict with Rachel's recent PR?

Reply:
I believe it builds upon it, unless I've missed something

Reply:
Her [PR](https://github.com/fleetdm/fleet/pull/14086/files) was using `first-letter`. Doesn't this override that?